src/services/resource.js

```diff
@@ -210,6 +327,394 @@ module.exports = class resourceHelper {
+				}
+				// if the organization have any resource type in sequential review type
+				if (resourceTypesInSequentialReview.length > 0) {
+					// fetch all sequential resource ids from org which are open to all
+					const sequentialResourcesIds = await this.findSequentialResources(
+						organization_id,
+						roles,
+						resourceTypesInSequentialReview
+					)
+					// add the resource ids in the final array
+					finalResourceIds = [...finalResourceIds, ...sequentialResourcesIds]
+
+					// fetch all resource ids assigned to another reviewer or reviewing by another reviewer
+					resourceIdsToBeRemoved = await this.findResourcesPickedUpByAnotherReviewer(
+						user_id,
+						finalResourceIds
+					)
+				}
+				// if the organization have any resource type in parallel review type
+				if (resourceTypesInParallelReview.length > 0) {
+					// fetch all parallel resource ids from org which is open to all
+					const parallelResourcesIds = await this.findParallelResources(
+						organization_id,
+						resourceTypesInParallelReview
+					)
+					// add the resource ids in the final array
+					finalResourceIds = [...finalResourceIds, ...parallelResourcesIds]
+				}
+
+				const assignedToMe = await this.findResourcesAssignedToReviewer(uniqueOrganizationIds, user_id)
+
+				finalResourceIds = [...finalResourceIds, ...assignedToMe]
+
+				finalResourceIds = _.difference(
+					utils.getUniqueElements(finalResourceIds),
+					utils.getUniqueElements(resourceIdsToBeRemoved)
+				)
+			}
+
+			if (finalResourceIds.length === 0) {
+				return responses.successResponse({
+					statusCode: httpStatusCode.ok,
+					message: 'RESOURCE_LISTED_SUCCESSFULLY',
+					result: [],
+				})
+			}
+
+			const resourceFilter = {
+				id: { [Op.in]: finalResourceIds },
+				organization_id: { [Op.in]: uniqueOrganizationIds },
+				user_id: {
+					[Op.notIn]: [user_id],
+				},
+			}
+			if (searchText != '')
+				resourceFilter.title = {
+					[Op.iLike]: '%' + searchText + '%',
+				}
+
+			if (common.TYPE in queryParams && queryParams[common.TYPE]) {
+				resourceFilter.type = queryParams[common.TYPE].split(',')
+			}
+
+			const response = await resourceQueries.resourceList(
+				resourceFilter,
+				[
+					'id',
+					'title',
+					'type',
+					'organization_id',
+					'status',
+					'user_id',
+					'submitted_on',
+					'last_reviewed_on',
+					'created_at',
+					'meta',
+				],
+				sort,
+				page,
+				limit
+			)
+
+			if (response.length === 0) {
+				return responses.successResponse({
+					statusCode: httpStatusCode.ok,
+					message: 'RESOURCE_LISTED_SUCCESSFULLY',
+					result: [],
+				})
+			}
+
+			const uniqueCreatorIds = utils.getUniqueElements(
+				response.result.map((item) => {
+					return item.user_id
+				})
+			)
+
+			const userDetails = await this.fetchUserDetails(uniqueCreatorIds)
+			const orgDetails = await this.fetchOrganizationDetails(uniqueOrganizationIds)
+
+			result.data = response.result.map((item) => {
```

Comment:
@adithyadinesh0412  handle it the result is empty

Reply:
if the result is empty , it will return an empty object and the final response will not have any organization details. i.e organization will be empty. We cannot restrict the response bec, even if we don't have the org / user details we can provide list.  @priyanka-TL